docs/readme-linux.md

```diff
@@ -0,0 +1,25 @@
+# Building on Linux
+
+You will need at least CMake 3.21. You can get the latest version from the [offical website](https://cmake.org/download/). You will also need a C++ compiler (usually gcc) and Ninja.
+
+**Note**: If your distro does not provide up-to-date version of the needed libraries, you will need to tell CMake to build the libraries from source by passing `-DES_USE_SYSTEM_LIBRARIES=OFF` to CMake when configuring.
+
+If you use a reasonably up-to-date distro, then you can use your favorite package manager to install the needed dependencies:
+
+<details>
+<summary>DEB-based distros</summary>
+
+```
+g++ cmake ninja-build libsdl2-dev libpng-dev libjpeg-dev libgl1-mesa-dev libglew-dev libopenal-dev libmad0-dev uuid-dev
+```
+
+</details>
+
+<details>
+<summary>RPM-based distros</summary>
+
+```
+gcc-c++ cmake ninja-build SDL2-devel libpng-devel libjpeg-turbo-devel mesa-libGL-devel glew-devel openal-soft-devel libmad-devel libuuid-devel
+```
+
+</details>
```

Comment:
Maybe also include arch package names? It would be something like
> gcc make ninja sdl2 libpng libjpeg-turbo mesa glew openal libmad util-linux-libs

I think these are the ones used on my distro, but do double-check them.